[2025-07-05T12:41:36.068+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-07-05T12:41:36.080+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: practice_etl_pipeline.transform_and_load manual__2025-07-05T12:41:27.100179+00:00 [queued]>
[2025-07-05T12:41:36.084+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: practice_etl_pipeline.transform_and_load manual__2025-07-05T12:41:27.100179+00:00 [queued]>
[2025-07-05T12:41:36.085+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-07-05T12:41:36.093+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): transform_and_load> on 2025-07-05 12:41:27.100179+00:00
[2025-07-05T12:41:36.096+0000] {clientserver.py:570} INFO - Closing down clientserver connection
[2025-07-05T12:41:36.097+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=7043) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-07-05T12:41:36.097+0000] {standard_task_runner.py:63} INFO - Started process 7185 to run task
[2025-07-05T12:41:36.099+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'practice_etl_pipeline', 'transform_and_load', 'manual__2025-07-05T12:41:27.100179+00:00', '--job-id', '94', '--raw', '--subdir', 'DAGS_FOLDER/main.py', '--cfg-path', '/tmp/tmp1h26m4mi']
[2025-07-05T12:41:36.101+0000] {standard_task_runner.py:91} INFO - Job 94: Subtask transform_and_load
[2025-07-05T12:41:36.133+0000] {task_command.py:426} INFO - Running <TaskInstance: practice_etl_pipeline.transform_and_load manual__2025-07-05T12:41:27.100179+00:00 [running]> on host 6802b12b52ea
[2025-07-05T12:41:36.181+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='practice_etl_pipeline' AIRFLOW_CTX_TASK_ID='transform_and_load' AIRFLOW_CTX_EXECUTION_DATE='2025-07-05T12:41:27.100179+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-07-05T12:41:27.100179+00:00'
[2025-07-05T12:41:36.181+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-07-05T12:41:36.760+0000] {clientserver.py:529} INFO - Error while sending or receiving.
Traceback (most recent call last):
  File "/opt/spark/spark-4.0.0-bin-hadoop3/python/lib/py4j-0.10.9.9-src.zip/py4j/clientserver.py", line 527, in send_command
    self.socket.sendall(command.encode("utf-8"))
ConnectionResetError: [Errno 104] Connection reset by peer
[2025-07-05T12:41:36.762+0000] {clientserver.py:570} INFO - Closing down clientserver connection
[2025-07-05T12:41:36.763+0000] {java_gateway.py:1056} INFO - Exception while sending command.
Traceback (most recent call last):
  File "/opt/spark/spark-4.0.0-bin-hadoop3/python/lib/py4j-0.10.9.9-src.zip/py4j/clientserver.py", line 527, in send_command
    self.socket.sendall(command.encode("utf-8"))
ConnectionResetError: [Errno 104] Connection reset by peer

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/spark/spark-4.0.0-bin-hadoop3/python/lib/py4j-0.10.9.9-src.zip/py4j/java_gateway.py", line 1038, in send_command
    response = connection.send_command(command)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/spark/spark-4.0.0-bin-hadoop3/python/lib/py4j-0.10.9.9-src.zip/py4j/clientserver.py", line 530, in send_command
    raise Py4JNetworkError(
py4j.protocol.Py4JNetworkError: Error while sending
[2025-07-05T12:41:36.765+0000] {clientserver.py:570} INFO - Closing down clientserver connection
[2025-07-05T12:41:37.523+0000] {logging_mixin.py:188} INFO - +--------------------+
|                 _c0|
+--------------------+
|31.12.2017;395775...|
|31.12.2017;15423;...|
|31.12.2017;298332...|
|31.12.2017;17434;...|
|31.12.2017;310405...|
|31.12.2017;22798;...|
|31.12.2017;13800;...|
|31.12.2017;13905;...|
|31.12.2017;40266;...|
|31.12.2017;341567...|
|31.12.2017;14912;...|
|31.12.2017;13871;...|
|31.12.2017;364395...|
|31.12.2017;14138;...|
|31.12.2017;17441;...|
|31.12.2017;18164;...|
|31.12.2017;14878;...|
|31.12.2017;341569...|
|31.12.2017;14913;...|
|31.12.2017;18007;...|
+--------------------+
only showing top 20 rows
[2025-07-05T12:41:38.643+0000] {logging_mixin.py:188} INFO - +----------+------+--------------------+-------+
| timestamp|status|             message|obj_num|
+----------+------+--------------------+-------+
|2025-07-05|ОШИБКА|ошибка перелива д...|    115|
+----------+------+--------------------+-------+
[2025-07-05T12:41:39.009+0000] {logging_mixin.py:188} INFO - +--------------------+
|                 _c0|
+--------------------+
|09-01-2018;873663...|
|09-01-2018;176169...|
|09-01-2018;409685...|
|09-01-2018;156987...|
|09-01-2018;120483...|
|09-01-2018;311159...|
|10-01-2018;873663...|
|10-01-2018;393808...|
|10-01-2018;37051;...|
|10-01-2018;13630;...|
|10-01-2018;37051;...|
|10-01-2018;176270...|
|10-01-2018;393808...|
|11-01-2018;13630;...|
|11-01-2018;434141...|
|11-01-2018;311161...|
|11-01-2018;37051;...|
|11-01-2018;176169...|
|12-01-2018;393808...|
|12-01-2018;434141...|
+--------------------+
only showing top 20 rows
[2025-07-05T12:41:39.439+0000] {logging_mixin.py:188} INFO - +----------+------+--------------------+-------+
| timestamp|status|             message|obj_num|
+----------+------+--------------------+-------+
|2025-07-05|ОШИБКА|ошибка перелива д...|  33893|
+----------+------+--------------------+-------+
[2025-07-05T12:41:39.610+0000] {logging_mixin.py:188} INFO - +--------------------+
|                 _c0|
+--------------------+
|2018-01-01;2018-0...|
|2018-01-01;2018-0...|
|2018-01-01;2018-0...|
|2018-01-01;2018-0...|
|2018-01-01;2018-0...|
|2018-01-01;2018-0...|
|2018-01-01;2018-0...|
|2018-01-01;2018-0...|
|2018-01-01;2018-0...|
|2018-01-01;2018-0...|
|2018-01-01;2018-0...|
|2018-01-01;2018-0...|
|2018-01-01;2018-0...|
|2018-01-01;2018-0...|
|2018-01-01;2018-0...|
|2018-01-01;2018-0...|
|2018-01-01;2018-0...|
|2018-01-01;2018-0...|
|2018-01-01;2018-0...|
|2018-01-01;2018-0...|
+--------------------+
only showing top 20 rows
[2025-07-05T12:41:39.892+0000] {logging_mixin.py:188} INFO - +----------+------+--------------------+-------+
| timestamp|status|             message|obj_num|
+----------+------+--------------------+-------+
|2025-07-05|ОШИБКА|ошибка перелива д...|    113|
+----------+------+--------------------+-------+
[2025-07-05T12:41:40.074+0000] {logging_mixin.py:188} INFO - +--------------------+
|                 _c0|
+--------------------+
|37;2017-05-11;205...|
|40;2017-05-11;205...|
|529511969;2017-12...|
|CURRENCY_RK;DATA_...|
|63;2017-05-11;205...|
|26;2017-05-11;205...|
|29;2017-05-11;205...|
|38;2017-05-11;205...|
|46;2017-05-11;205...|
|35;2017-05-11;205...|
|39;2017-05-11;205...|
|30;2017-05-11;205...|
|48;2017-05-11;205...|
|52;2017-05-11;205...|
|56;2017-05-11;205...|
|59;2017-05-11;205...|
|60;2017-05-11;205...|
|32;2017-05-11;205...|
|23;2017-05-11;205...|
|25;2017-05-11;205...|
+--------------------+
only showing top 20 rows
[2025-07-05T12:41:40.339+0000] {logging_mixin.py:188} INFO - +----------+------+--------------------+-------+
| timestamp|status|             message|obj_num|
+----------+------+--------------------+-------+
|2025-07-05|ОШИБКА|ошибка перелива д...|     51|
+----------+------+--------------------+-------+
[2025-07-05T12:41:40.481+0000] {logging_mixin.py:188} INFO - +--------------------+
|                 _c0|
+--------------------+
|2018-01-30;2018-0...|
|2018-01-19;2018-0...|
|2018-01-10;2018-0...|
|2018-01-12;2018-0...|
|2018-01-12;2018-0...|
|2018-01-27;2018-0...|
|2018-01-10;2018-0...|
|2018-01-10;2018-0...|
|2018-01-25;2018-0...|
|2018-01-20;2018-0...|
|2018-01-13;2018-0...|
|DATA_ACTUAL_DATE;...|
|2018-01-17;2018-0...|
|2018-01-20;2018-0...|
|2018-01-18;2018-0...|
|2018-01-16;2018-0...|
|2018-01-12;2018-0...|
|2018-01-11;2018-0...|
|2017-12-30;2018-0...|
|2018-01-23;2018-0...|
+--------------------+
only showing top 20 rows
[2025-07-05T12:41:40.746+0000] {logging_mixin.py:188} INFO - +----------+------+--------------------+-------+
| timestamp|status|             message|obj_num|
+----------+------+--------------------+-------+
|2025-07-05|ОШИБКА|ошибка перелива д...|    461|
+----------+------+--------------------+-------+
[2025-07-05T12:41:40.877+0000] {logging_mixin.py:188} INFO - +--------------------+
|                 _c0|
+--------------------+
|А;Балансовые счет...|
|А;Балансовые счет...|
|А;Балансовые счет...|
|А;Балансовые счет...|
|А;Балансовые счет...|
|А;Балансовые счет...|
|А;Балансовые счет...|
|А;Балансовые счет...|
|А;Балансовые счет...|
|CHAPTER;CHAPTER_N...|
|А;Балансовые счет...|
|А;Балансовые счет...|
|А;Балансовые счет...|
|А;Балансовые счет...|
|А;Балансовые счет...|
|А;Балансовые счет...|
|А;Балансовые счет...|
|А;Балансовые счет...|
|А;Балансовые счет...|
+--------------------+
[2025-07-05T12:41:41.128+0000] {logging_mixin.py:188} INFO - +----------+------+--------------------+-------+
| timestamp|status|             message|obj_num|
+----------+------+--------------------+-------+
|2025-07-05|ОШИБКА|ошибка перелива д...|     19|
+----------+------+--------------------+-------+
[2025-07-05T12:41:41.129+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-07-05T12:41:41.130+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-07-05T12:41:41.135+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=practice_etl_pipeline, task_id=transform_and_load, run_id=manual__2025-07-05T12:41:27.100179+00:00, execution_date=20250705T124127, start_date=20250705T124136, end_date=20250705T124141
[2025-07-05T12:41:41.147+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-07-05T12:41:41.157+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-07-05T12:41:41.160+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
